docroot/themes/custom/uids_base/scss/paragraphs/uiowa-paragraphs-lists.scss

```diff
@@ -659,6 +367,7 @@
 }
 
 // IE 11
+// @todo remove since we are no longer support IE11?
```

Comment:
Should we keep these and just not provide _additional_ support for IE11?

Reply:
I think these would need to be rewritten, since the structure of the card changed. So, leaving them as is might have unintended consequences. Without testing them, its hard to know. Removing seems like the right move to me.

Reply:
i am going to test this quick in browserstack real quick

Reply:
With:
<img width="1187" alt="Screenshot 2023-03-24 at 11 27 06 AM" src="https://user-images.githubusercontent.com/4663676/227585599-86dd8bf3-3a4d-4482-a867-ce868783c1fc.png">
Without:
<img width="1094" alt="Screenshot 2023-03-24 at 11 29 09 AM" src="https://user-images.githubusercontent.com/4663676/227585608-d7704cb3-4c89-47cd-ab1b-82b918608ff3.png">

Keeping the styles around for now.